tasks/configure-directories.yml

```diff
@@ -0,0 +1,28 @@
+---
```

Comment:
It might be worth combining these into a single task using `with_items `. I've done similar [here](https://github.com/companieshouse/kafka-ami/blob/main/ansible/roles/kafka/tasks/install_kafka.yml#L68-L83)